source/guides/helpers/forms.md

```diff
@@ -201,6 +203,140 @@ Developers can customize attack handling.
 </form>
 ```
 
+### Pass A Value
```

Comment:
yes, no problem. I added this part because I am often asked about share form code between views. What you think can we do something to help with this issue?